client/state/action-types.js

```diff
@@ -61,4 +61,8 @@ export const SUPPORT_USER_TOKEN_FETCH = 'SUPPORT_USER_TOKEN_FETCH';
 export const SUPPORT_USER_TOKEN_SET = 'SUPPORT_USER_TOKEN_SET';
 export const SUPPORT_USER_RESTORE = 'SUPPORT_USER_RESTORE';
 export const USER_RECEIVE = 'USER_RECEIVE';
+export const VIDEO_FETCH = 'VIDEO_FETCH';
```

Comment:
>I do not feel the need to have this distinction in the language, and for me `REQUEST` covers both scenarios _[...]_ I am happy to agree on the term `REQUEST` and would prefer using that in my action names.

Cool, I'd tend to agree. Unrelated to this pull request, we'll want to see to adopting this consistency with the other action types, and investigating how well this will fit with non-retrieval requests (e.g. `POST_UPDATE_REQUEST` for saving post?)